cypress/cypress/e2e/submariner_install.cy.js

```diff
@@ -0,0 +1,44 @@
+/** *****************************************************************************
+ * Licensed Materials - Property of Red Hat, Inc.
+ * Copyright (c) 2021 Red Hat, Inc.
+ ****************************************************************************** */
+
+/// <reference types="cypress" />
+
+import { submarinerClusterSetMethods } from '../views/submariner/actions/submariner_actions'
+import { clusterSetMethods } from '../views/clusterset/clusterset'
+
+describe('submariner - Deployment validation', {
+    tags: ['@submariner', '@e2e'],
```

Comment:
Please, remove the '@e2e' tag until we will verify this test within the working pipeline.